src/ontology/fbbt-edit.obo

```diff
@@ -123798,6 +123795,19 @@ intersection_of: part_of FBbt:00003701 ! adult optic lobe
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-06-25T14:07:29Z
 
+[Term]
+id: FBbt:00058227
+name: lobula-lobula plate columnar neuron LLPC1
+def: "A lobula-lobula plate columnar neuron that receives input from layer 4 of the lobula and layers 1 and 3 of the lobula plate (Isaacson, 2018; Davis et al., 2020). It is highly sensitive to front-to-back motion on the ipsilateral eye (Isaacson, 2018). It is cholinergic (Davis et al., 2020)." [doi:10.17863/CAM.36002, FlyBase:FBrf0244935]
```

Comment:
year for thesis should be 2019? https://www.repository.cam.ac.uk/handle/1810/288741

Reply:
I used the date from the thesis file, I didn't notice the date in the repository was different… I agree that using the date from the repository makes more sense, since that's how the thesis was registered.